### PR TITLE
docs(aio): change Stack Overflow link

### PR DESCRIPTION
**Pull request by sarunint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/navigation.json [hunk 1]
```diff
@@ -424,7 +424,7 @@
       "title": "Help",
       "children": [
         {
-          "url": "http://stackoverflow.com/questions/tagged/angular2",
+          "url": "http://stackoverflow.com/questions/tagged/angular",
```
Comment (gkalpak) [inline aio/content/navigation.json]: 👍
Could you also change `http` to `https`?

**Review by gkalpak — APPROVED** [review]

**Comment by sarunint** [comment]
Oh, I didn't notice that.

Will fix asap

**Comment by matsko** [comment]
Landed as https://github.com/angular/angular/commit/290b0b48a93eb6fb9e1bb6754f4a379ac899f8ce and https://github.com/angular/angular/commit/bd04cd61f842e63164ddecd46098268e9f4a3ac0

**Comment by matsko** [comment]
Thank you @sarunint

**Pull request closed without merging**